sdk/core/AzureCore/Source/ClientLogger.swift

```diff
@@ -156,15 +156,15 @@ public class OSLogger: ClientLogger {
 
     // MARK: Initializers
 
-    public init(withLogger osLogger: OSLog, level: ClientLogLevel = .info) {
+    public init(withLogger osLogger: OSLog, atLevel level: ClientLogLevel = .info) {
         self.level = level
         self.osLogger = osLogger
     }
 
     public init(
-        subsystem: String = "com.azure",
-        category: String,
-        level: ClientLogLevel = .info
+        forSubsystem subsystem: String = "com.azure",
+        withCategory category: String,
+        atLevel level: ClientLogLevel = .info
```

Comment:
Any specific reason for this change?

Reply:
No, just that this was the first swift code I wrote, and I probably didn't understand the whole internal vs external name thing. I think this is what I would have called it if I'd had the benefit of hindsight that I do now.

Reply:
It just seems that the old style was more consistent with the internal implementation classes:
```Swift
OSLog(subsystem: subsystem, category: category)
NSLogger(tag: "MyAppClient", level: .debug)
```

Reply:
I don't have a strong opinion, I guess the underlying logger APIs don't really feel particularly "swifty" either. But maybe you're right, better to be consistent with the existing Foundation classes even if they feel less idiomatic. I'll remove that change.